Collection/SchemaDefinitionCollection.php

```diff
@@ -13,14 +13,13 @@
 
 namespace Linkin\Bundle\SwaggerResolverBundle\Collection;
 
+use function array_values;
```

Comment:
@adrenalinkin all `use function` are removed and it uses the  Symfony standards. `\` are added if needed. Because you metiond the support of symfony 3  in #72 I downgraded to `friendsofphp/php-cs-fixer@2.19.3`